tokio/src/sync/watch.rs

```diff
@@ -114,7 +114,7 @@
 use crate::sync::notify::Notify;
 
 use crate::loom::sync::atomic::AtomicUsize;
-use crate::loom::sync::atomic::Ordering::Relaxed;
+use crate::loom::sync::atomic::Ordering::*;
```

Comment:
You don't change anything outside of `mod state`, so this doesn't seem necessary.